.travis.yml

```diff
@@ -34,11 +34,18 @@ install:
   - cd woocommerce/
   - git checkout tags/$WOO_VERSION
   - bash tests/bin/install.sh woocommerce_test root '' localhost $WP_VERSION
+  - cd /tmp
+  - mkdir /tmp/tmp1
+  - wget https://github.com/SwedbankPay/swedbank-pay-woocommerce-payments/archive/master.zip
```

Comment:
Can't `swedbank-pay-woocommerce-payments` be installed with composer instead?

Reply:
![Screenshot](https://user-images.githubusercontent.com/6286270/66398763-2ac58c80-ea00-11e9-9c5a-a44c645beb6c.png)

Reply:
Yes, but can't we get that published on Packagist first, then add it as a Composer requirement here after?

Reply:
Yes, will do